build/commands/lib/config.js

```diff
@@ -540,6 +540,12 @@ Config.prototype.buildArgs = function () {
   } else {
     args.cc_wrapper = path.join(this.srcDir, 'brave', 'script', 'redirect-cc.py')
   }
+
+  // Limit action pool (non-compile actions) to amount of CPU cores.
```

Comment:
this should go in `let args = {`

Reply:
the `args.* = ...` stuff is for things that override the default values set it `let args`

Reply:
yep, makes sense. moved to `let args` block.